README.md

```diff
@@ -1,3 +1,97 @@
-# tsuru-performance-tests
+# Tsuru Performance test suite
+
+The Tsuru test suite is a set of jmeter-ruby-dsl scripts describing a set of performance testing scenarios.
+They each can be found in the `dsl` folder and run locally or exported to jmx files for running on Blazemeter,
+Flood.io or on your own servers, to provide more resources to a performance test.
+
+Currently, these tests are only runnable against the CI environment.
+
+## How to install
+
+```
+Brew install jmeter
+git clone https://github.com/alphagov/tsuru-performance-testing
+cd ./tsuru-performance-testing
+bundle install
+```
+
+## How to run these tests locally
+
+To run the tests:
+
+```
+cd  ./dsl
+bundle exec ruby deploy_apps.rb <TSURU_ENVIRONMENT_NAME>  <TSURU_TARGET_HOST> 100 100
+```
+
+This will generate a jmeter.jmx file which you can load into the Jmeter gui or use in a headless mode.
```

Comment:
The ruby jmeter dsl offers a headless mode; currently we don't use it because of the os process sampler issue. This is a good alternative though to having to run the jmeter GUI - I've updated the README.md to include this option for the deploy_apps.rb script (as that is the only one that can't be run in the correct headless method).